lib/patch-dom.js

```diff
@@ -63,8 +71,8 @@ export default ({oldArticleElm, newArticleElm}) => {
       case UPDATE:
         if (isIframeEmbed(prev)) {
           queue.update.push(() => updateIframeEmbed({
-            oldFigure: oldArticleElm.childNodes[pos],
-            newFigure: next
+            oldFigure: oldArticleElm.children[pos],
```

Comment:
Hm, ok strange. Is flow not complaining about the other places we're using childNodes in here? 

It's probably fine to change this here, since we actually are looking for an element and not a node. Not sure our tests are covering this a 100% though so we should just make sure manually that this is working as expected.